Sources/Data/models/Model.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>Model15.xcdatamodel</string>
+	<string>Model16.xcdatamodel</string>
```

Comment:
We have another database migration pending with the Shared Playlist folders, we have to make sure the DB is in sync.
Probably need two migrations to make it correct.

What did you add or updated in the database btw?

Reply:
An extra table. Do you think I need a migration for this?

Reply:
yeah you do, depending which code merges first we have to adjust the database on the remaining opened PR